### ISSUE TITLE
Mailing list for comments

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wcag2ict#729]
In our [Status of this Document section](https://w3c.github.io/wcag2ict/#sotd)

> [...] If filing issues in GitHub is not feasible, send email to public-agwg-comments@w3.org (comment archive).

Not sure when or why we changed to the AGWG list, but we have a [public-wcag2ict-comments](https://lists.w3.org/Archives/Public/public-wcag2ict-comments/) mailing list that is the one we should be using.
